gcloud-java-gax/README.md

```diff
@@ -0,0 +1,55 @@
+Google Cloud Java Client
+==========================
+
+Java idiomatic client for [Google Cloud Platform][cloud-platform] services.
+
+[![Build Status](https://travis-ci.org/GoogleCloudPlatform/gcloud-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/gcloud-java)
+[![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/gcloud-java/badge.svg?branch=master)](https://coveralls.io/r/GoogleCloudPlatform/gcloud-java?branch=master)
+
+-  [Homepage] (https://googlecloudplatform.github.io/gcloud-java/)
+-  [API Documentation] (http://googlecloudplatform.github.io/gcloud-java/apidocs)
+-  [Examples] (http://googlecloudplatform.github.io/gcloud-java/apidocs/index.html?com/google/gcloud/examples/package-summary.html)
+
+This module provides common functionality and is required by the other service specific modules.
+
+Quickstart
+----------
+Add this to your pom.xml file
+```xml
+<dependency>
+  <groupId>com.google.gcloud</groupId>
+  <artifactId>gcloud-java-gax</artifactId>
+  <version>0.0.6</version>
```

Comment:
We are currently at 0.0.10, but more importantly, can you get this README.md based on a more recent state of [this](https://github.com/GoogleCloudPlatform/gcloud-java/tree/master/gcloud-java-core)?